src/components/ProjectList/CardsContainer.jsx

```diff
@@ -119,16 +119,18 @@ export default class CardsContainer extends React.Component {
       <div>
         <div id='container'>
           <div className='inputContainer'>
-            <input id='search' type='text' name='search' placeholder='Search...' onChange={this.handleChange} />
+            <input id='search' type='text' name='search' placeholder='Search...' onChange={this.handleChange} aria-label='Search'/>
           </div>
-          <div className='inputContainer'>
+          <div id="tag-selector-container" className='inputContainer'>
             <Select
               name='tag-selector'
               value={this.state.value}
               onChange={this.handleSelectChange}
               options={this.filterOptions}
               multi={true}
-              placeholder='Filter'
+              placeholder={<div className='filter-placeholder-text' style={{color: "#757575"}}>Filter</div>}
```

Comment:
Could you use class instead of style for Select component?

Reply:
Did this for the placeholder text and then removed the other Select styling since this was not actually affecting the accessibility!